database/gormdb_assignment.go

```diff
@@ -34,7 +34,7 @@ func (db *GormDB) CreateAssignment(assignment *pb.Assignment) error {
 		Assign(map[string]interface{}{
 			"name":              assignment.Name,
 			"order":             assignment.Order,
-			"language":          assignment.Language,
+			"language":          assignment.ScriptFile,
```

Comment:
Shouldn't it be `"script_file": assignment.ScriptFile`? Now it attempts to save to the `language` field of the `Assignment` table, which should not even exist.

Reply:
Yes, thanks! That's a mistake. Will fix